src/main/scala/tech/cryptonomic/conseil/routes/AppInfo.scala

```diff
@@ -0,0 +1,29 @@
+package tech.cryptonomic.conseil.routes
+
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.marshalling.{PredefinedToEntityMarshallers, ToEntityMarshaller}
+import akka.http.scaladsl.model.MediaTypes
+import akka.http.scaladsl.server.Directives._
+
+import tech.cryptonomic.conseil.BuildInfo
+import tech.cryptonomic.conseil.util.JsonUtil.{toJson, JsonString}
+
+/** defines endpoints to expose the currently deployed application information (e.g version, ...) */
+object AppInfo {
+
+  /** data type collecting relevant information to expose */
+  case class Info(application: String, version: String)
+
+  //add the correct content-type for [[JsonUtil]]-converted values
+  implicit private val jsonMarshaller: ToEntityMarshaller[JsonString] =
+    PredefinedToEntityMarshallers.StringMarshaller
+      .compose((_: JsonString).json)
+      .wrap(MediaTypes.`application/json`)(identity _)
+
+  /** the endpoints to expose application information through http */
+  val route: Route = pathEnd {
+    get {
+      complete(toJson(Info(application = BuildInfo.name, version = BuildInfo.version)))
```

Comment:
Sorry if I'm missing anything obvious but is the `BuildInfo` file checked in somewhere?

Reply:
It's perfectly understandable, it's generated from the 
`"com.eed3si9n" % "sbt-buildinfo" % "0.9.0"` plugin when it loads the build

It is also customized in the build file using the following lines
```
//add build information as an object in code
enablePlugins(BuildInfoPlugin)
buildInfoKeys := Seq[BuildInfoKey](name, version, scalaVersion, sbtVersion)
buildInfoPackage := "tech.cryptonomic.conseil"
```